content/ja/handbook/engineering/data-engineering/analytics/analytics-instrumentation/index.md

```
@@ -0,0 +1,606 @@
+| ラベル | 使用方法 |
+| -- | -- |
+| `~"workflow::validation backlog"` | まだ精緻化も優先順位付けもされていない受信 Issue に対してプロダクトマネージャーが適用。 |
+| `~"workflow::problem validation"` | PM が問題を徹底的に理解している Issue に対してプロダクトマネージャーが適用。 |
+| `~"workflow::design"` | ソリューションをアイデア出し・提案するためにプロダクトマネージャーまたはデザイナー（または Analytics Instrumentation エンジニア）が適用。提案されたソリューションは技術的な実現可能性を確認するためにエンジニアリングによってレビューされるべきです。 |
+| `~"workflow::solution validation"` | ユーザーインタビューやユーザビリティテストを通じて提案されたソリューションを検証するためにプロダクトマネージャーまたはデザイナー（または Analytics Instrumentation エンジニア）が適用。 |
+
+#### ビルドステージ
+
+| ラベル | 使用方法 |
+|--|--|
+| `~"workflow::planning breakdown"` | エンジニアが Issue を分解して[見積もりを追加する](#estimation)ために、プロダクトマネージャーが適用。 |
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)